app/controllers/async_info_controller.rb

```diff
@@ -29,7 +29,7 @@ def shell_version
     set_surrogate_key_header "shell-version-endpoint"
     # shell_version will change on every deploy.
     # *Technically* could be only on changes to assets and shell, but this is more fool-proof.
-    shell_version = ApplicationConfig["RELEASE_FOOTPRINT"] + SiteConfig.admin_action_taken_at.to_s
+    shell_version = ForemInstance.deployed_at + SiteConfig.admin_action_taken_at.to_s
```

Comment:
Using the ForemInstance.deployed_at method here as will set ApplicationConfig["RELEASE_FOOTPRINT"] if it exists.